pkgs/applications/misc/calibre/default.nix

```
@@ -42,9 +43,20 @@ mkDerivation rec {
     # the unrar patch is not from debian
   ] ++ lib.optional (!unrarSupport) ./dont_build_unrar_plugin.patch;
 
+  escaped_pyqt5_dir = builtins.replaceStrings ["/"] ["\\/"] (toString python3Packages.pyqt5);
+  platform_tag =
+    if stdenv.hostPlatform.isDarwin then
+      "\\\"WS_MACX\\\""
+    else if stdenv.hostPlatform.isWindows then
+      "\\\"WS_WIN\\\""
+    else
+      "\\\"WS_X11\\\"";
```

```suggestion
     ''"WS_MACX"''
    else if stdenv.hostPlatform.isWindows then
      ''"WS_WIN"''
    else
      ''"WS_X11"'';
```

or do we need the \"?

---

We need the `\"`, I changed it to `''\"WS_X11\"''`.